src/main/java/uk/gov/justice/digital/delius/transformers/StaffTransformer.java

```diff
@@ -58,6 +59,7 @@ public static StaffHuman staffOf(Staff staff) {
             .forenames(combinedMiddleNamesOf(staff.getForename(), staff.getForname2()))
             .surname(staff.getSurname())
             .code(staff.getOfficerCode())
+            .staffGrade(staff.getGrade()!=null?staff.getGrade().getCodeValue():null)
```

Comment:
Nitpick: Add a few spaces between the operators?

Reply:
ok